test/integration/test/ip_camera_test.js

```diff
@@ -5,12 +5,14 @@ const Config = require('config-js');
 const isoDate = require('iso-date');
 
 var config = new Config('./test/integration/lib/config/foxbox.js');
-var Prepper = require('../lib/testPrepper.js');
+var Prepper = require('../lib/make_suite.js');
 
 Prepper.makeSuite('Control camera locally',function(){
 
   var cameraService;
 
+  before(Prepper.turnAllSims);
```

Comment:
Having many `before()`s is undocumented. Do you have a link to demonstrate that mocha is supposed to run all the befores in the correct order?

Otherwise, I'd stick with something like:
```js
before(() => {
  return Prepper.turnOnAllSimulators()
    .then(() => Prepper.turnOnFoxbox())
    .then(() => Prepper.foxboxManager.foxboxLogin())
});
```

That makes me realize `turnOnAllSimulators` is suppossedly sync, `turnOnFoxbox()` expects a callback and `foxboxLogin()` is a Promise. I don't know how many `before()`s deal with that particular case. I'd make what needs to be async returns promises